Post-release (0.2.0) dependency updates & related

eyelidlessness opened the pull request:
Not totally fuss-free, but less fuss than the last one!

More details in commit notes.

sadiqkhoja reviewed (approved):
Tried this PR locally, all seems to be working!

eyelidlessness commented:
Thanks for a first look @sadiqkhoja! Since you mentioned the conflict with #162, I just went ahead and reverted the `scripts` package removal and added a no-op `test` script to it. That way it won't cause similar hassles next time we do project wide dependency updates.

If you're still good with the change, please feel free to merge!
